web/source/keyboards/kmwkeyboards.ts

```diff
@@ -893,30 +887,51 @@ namespace com.keyman.keyboards {
       }
     };
 
+    /**
+     * Return a Promise of the merged keyboard stubs and error stubs
+     * @param {(KeyboardStub|ErrorStub)[]} keyboardStubs 
+     * @param {ErrorStub[]} errorStubs 
+     * @returns {Promise<KeyboardStub|ErrorStub)[]>}
+     */
```

Comment:
Don't put types into the doc comment -- typescript already has those. Just descriptions of the parameters and return values

```suggestion
    /**
     * Returns a Promise of the merged keyboard stubs and error stubs.
     * 
     * If the keyboard stub array is empty, will return a rejected Promise, 
     * otherwise returns a resolved Promise.
     *
     * @param keyboardStubs  array of keyboard stubs to merge.
     * @param errorStubs     array of error stubs to merge.
     * @returns  resolved or rejected promise with merged array of stubs.
     */
```